Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -246,11 +246,12 @@ void WebFullScreenManager::enterFullScreenForElement(WebCore::Element* element,
 
         auto mimeType = emptyString();
         if (auto* cachedImage = renderImage->cachedImage()) {
-            if (auto* image = cachedImage->image())
+            if (auto* image = cachedImage->image()) {
                 mimeType = image->mimeType();
 
-            if (!MIMETypeRegistry::isSupportedImageMIMEType(mimeType))
-                mimeType = MIMETypeRegistry::mimeTypeForPath(cachedImage->url().string());
+                if (!MIMETypeRegistry::isSupportedImageMIMEType(mimeType))
+                    mimeType = MIMETypeRegistry::mimeTypeForExtension(image->filenameExtension());
+            }
```

Comment:
I think this change is not related to this change. Please remove and address it in separate PR.